pkg/monitoring/metrics/operator_metrics_test.go

```diff
@@ -9,20 +9,16 @@ import (
 
 var _ = Describe("Operator Metrics", func() {
 	Context("kubevirt_hco_system_health_status", func() {
-		It("should set system error reason correctly", func() {
-			metrics.SetHCOSystemError("Reason1")
-			v, err := metrics.GetHCOMetricSystemHealthStatus("Reason1")
+		It("should set the correct system health status", func() {
+			metrics.SetHCOMetricSystemHealthStatus(2)
```

Comment:
Please use the constant: `metrics.SystemHealthStatusError`

Reply:
it's the same thing. the constant is just a name for the value. but with the constant, the code is more readable.

Reply:
I mean what it will give me here:
 Expect(v).To(Equal(metrics.SystemHealthStatusError))

Reply:
The same. `metrics.SystemHealthStatusError` is 2

Reply:
```go
const (
	SystemHealthStatusHealthy float64 = iota
	SystemHealthStatusWarning
	SystemHealthStatusError
)
```
See here: https://go.dev/ref/spec#Iota

Reply:
sure, fixed thanks!